.github/workflows/integration.yml

```
@@ -43,15 +43,17 @@ on:
         options:
           - minimal
           - minimal-with-team
-          - monitoring-with-team
           - full
           - upgrade
+          - No-Apl
```

Nitpick but lets keep the casing consistent.
```suggestion
          - no-apl
```